recipes/compas/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "COMPAS" %}
+{% set version = "0.3.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "16b825123798359c3ed1a08afbf426ba20b1271c01d9561bea260c1dc698ce1a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
```

Comment:
Can this be `noarch: python`

https://conda-forge.org/docs/meta.html#building-noarch-packages

Reply:
Unfortunately not really `noarch`. This framework is basically a cross architecture tool, and chances are we'll add more and more arch-specific stuff in the near future.